docs/README.md

```diff
@@ -0,0 +1,25 @@
+# What is Happy Path?
+
+Happy Path, affectionately known internally as "Happy", is an opinionated project lifecycle management tool which helps you
+with the complexities of managing software deployments across multiple environments and AWS accounts. It is designed to
+set you up early on the Happy Path to infrastructure success from the beginning of your project, rather than needing to
+figure it out as an afterthought when it's least convenient.
+
+# Table of Contents
+
+## Getting Started
+
+* [Install Happy Path](Installing_Happy_Path.md)
+* [Create a Happy Environment](Creating_a_Happy_Environment.md)
```

Comment:
I can help fill out these sections in the next iteration

Reply:
Thank you! I'm a little stuck on some of the modules not being in the public repos. I've put the doc tickets into Blocked on the ticket to move the Happy TF modules. To Be Continued, I guess.